.github/workflows/publish-packages.yml

```diff
@@ -63,21 +63,17 @@ jobs:
       - name: Install PPA dependencies
         run: |
           sudo apt-get update
-          sudo apt-get install debhelper devscripts ubuntu-dev-tools -y --no-install-recommends
-      - name: Install Python
-        uses: actions/setup-python@v5
-        with:
-          python-version: 3.x
+          sudo apt-get install debhelper devscripts ubuntu-dev-tools -y
       - name: Install launchpadlib
         run: pip install launchpadlib --upgrade
       - name: Copy to other Ubuntu releases
-        env: 
+        env:
           LP_CREDENTIALS_FILE: /tmp/lp_credentials
         run: |
           echo ${{ secrets.PPA_COPY_TOKEN }} | base64 --decode > /tmp/lp_credentials
           git clone https://git.launchpad.net/ubuntu-archive-tools
           cd ubuntu-archive-tools
-          for release in "trusty" "bionic" "focal" "kinetic"
+          for release in "focal" "mantic"
```

Comment:
@FalcoXYZ Is it worth deploying for interim Ubuntu builds? They expire quickly and cause errors.

Reply:
What errors do they cause? I do agree removing the older ones though.

Reply:
for instance,
> kinetic is obsolete and will not accept new uploads.

and workflow fails because of that

Reply:
Should we mention somewhere in docs which distros do we support in PPA?
Does thosecover both amd and arm machines?